docs/narr/introduction.rst

```diff
@@ -859,14 +859,14 @@ Testing
 Every release of Pyramid has 100% statement coverage via unit and integration
 tests, as measured by the ``coverage`` tool available on PyPI.  It also has
 greater than 95% decision/condition coverage as measured by the
-``instrumental`` tool available on PyPI.  It is automatically tested by the
-Jenkins tool on Python 2.6, Python 2.7, Python 3.3, Python 3.4, Python 3.5,
-PyPy, and PyPy3 after each commit to its GitHub repository. Official Pyramid
-add-ons are held to a similar testing standard.  We still find bugs in Pyramid
-and its official add-ons, but we've noticed we find a lot more of them while
-working on other projects that don't have a good testing regime.
-
-Example: http://jenkins.pylonsproject.org/
+``instrumental`` tool available on PyPI. It is automatically tested by the
+Travis on Python 2.7, Python 3.3, Python 3.4, Python 3.5, PyPy, and PyPy3
+after each commit to its GitHub repository. Official Pyramid add-ons are held
+to a similar testing standard.  We still find bugs in Pyramid and its official
+add-ons, but we've noticed we find a lot more of them while working on other
+projects that don't have a good testing regime.
+
+Example: https://travis-ci.org/Pylons/pyramid
```

Comment:
@tseaver pointed out a failure on Jenkins as a result of a PR I once made, which passed on Travis-CI. I honestly don't know how Jenkins is used. Perhaps notifications?